Changelog.rst

```diff
@@ -3,6 +3,9 @@ version NEXT
 
 **2024-??-??**
 
+* Allow a halo to be added by `cf.Field.indices` and
+  `cf.Field.subspace`
+  (https://github.com/NCAS-CMS/cf-python/issues/759)
```

Comment:
I think that this might just confuse further, as you couldn't have provided the positional argument with a keyword in the first place:
```python
>>> def f(*mode, **kwargs):
...     print (mode, kwargs)
...
>>> f(mode=56)
() {'mode': 56}
```

Reply:
Oh fair, I didn't realise (though should have checked before raising the suggestion!) that we supply `mode` via `*args` and not `arg`, where in the latter case it could be supplied as a keyword. In which case, nothing to do and I will mark this as resolved.